nbconvert/templates/latex/article.tplx

```diff
@@ -1,10 +1,10 @@
 
-% Default to the notebook output style
+((=- Default to the notebook output style -=))
```

Comment:
Great catch!

Reply:
One thought is that might be too greedy though with the `-` since that consumes whitespace that would previously have been added by the `set cell_style`. I think in this case it makes sense, but we should be careful whenever adding or subtracting `-` to our delimiters.

Reply:
The latex comment has been annoying me for a long time. There are a few other places that contribute to an unnecessary amount of whitespace at the top of the generated latex file that also annoy me when working with the latex file (not that it really matters), but I figured that I would start with this one while I was in that file.